### ISSUE TITLE
Auto alignment for CJK fonts

### ISSUE THREAD

**Issue by 1900011604** [texstudio-org/texstudio#1336]
![Screenshot_20201103_191906](https://user-images.githubusercontent.com/65082432/97979045-7b790600-1e09-11eb-8d73-c840ea9e71de.png)

Can Texstudio import alignment for CJK fonts with Latin fonts like Konsole(Vim) and Windows Terminal?

**Comment by 1900011604** [comment]
That is, for **CJK width : Latin width < 1 : 1**, add more space to CJK fonts tp match 1:1.

**Comment by 1900011604** [comment]
![Screenshot_20201125_002455](https://user-images.githubusercontent.com/65082432/100122707-ab6d7380-2eb4-11eb-8928-d6b52ccb5267.png)

For Example: this is a Chinese bracket in two "$", the bracket has been put behind the "$"

$（$

**Comment by muzimuzhi** [comment]
@1900011604 Try the setting posted in https://github.com/texstudio-org/texstudio/issues/1099#issuecomment-632485580, which should fix the wrong-CJK-character-width problem.

**Comment by 1900011604** [comment]
I can confirm that choosing "disable fixed pitch mode" fixes things for the JetBrains Mono & MSYH font. Thanks!

**Comment by sunderme** [comment]
> \circ$（$t
> ![Screenshot_20201125_002455](https://user-images.githubusercontent.com/65082432/100122707-ab6d7380-2eb4-11eb-8928-d6b52ccb5267.png)
> 
> For Example: this is a Chinese bracket in two "$", the bracket has been put behind the "$"

It is generally not a good idea to put unrelated info/questions/reports in open or closed existing reports

**Comment by 1900011604** [comment]
I do not think it as unrelated, it is a very good example about unaligned characters
